stac_ipyleaflet/core.py

```diff
@@ -35,6 +28,9 @@ class StacIpyleaflet(Map):
     titiler_stac_endpoint = TITILER_STAC_ENDPOINT
 
     def __init__(self, **kwargs):
+        from stac_ipyleaflet.widgets.inspect import InspectControlWidget
+        from stac_ipyleaflet.widgets.draw import DrawControlWidget
```

Comment:
📓 : This helps to avoid those circular import errors and is better because instead of calling in the beginning of the file, is it just called in the function where it is needed